libraries/AP_Logger/LogStructure.h

```diff
@@ -1477,7 +1490,9 @@ Format characters in the format string for binary log messages
     { LOG_OPTFLOW_MSG, sizeof(log_Optflow), \
       "OF",   "QBffff",   "TimeUS,Qual,flowX,flowY,bodyX,bodyY", "s-EEEE", "F-0000" }, \
     { LOG_WHEELENCODER_MSG, sizeof(log_WheelEncoder), \
-      "WENC",  "Qfbfb", "TimeUS,Dist0,Qual0,Dist1,Qual1", "sm-m-", "F0-0-" }
+      "WENC",  "Qfbfb", "TimeUS,Dist0,Qual0,Dist1,Qual1", "sm-m-", "F0-0-" }, \
+    { LOG_ADSB_MSG, sizeof(log_ADSB), \
+      "ADSB",  "QIiiiHHhH", "TimeUS,ICAO_ADRS,LAT,LNG,ALT,HEADING,HOR_VEL,VER_VEL,SQUAWK", "s-DUmhnn-", "F-GGCBCC-" }
```

Comment:
I think this is too long. Have you tested it and looked at the log?

Reply:
works fine, I have uploaded a example log here, https://1drv.ms/u/s!Apevk2F1cJOqiiL490omfibANnLv first logging only special ID then everything